circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -129,7 +129,7 @@ constexpr size_t NUM_FIELDS_PER_SHA256 = 2;
  * |-----------+-------------------------------+----------------------|
  * | LOW       | n ≤ 8                         | 0 < hash_index ≤ 32  |
  * | MID       | 8 < n ≤ 16                    | 32 < hash_index ≤ 40 |
- * | HIGH      | 16 < n ≤ 44                   | 40 < hash_index ≤ 44 |
+ * | HIGH      | 16 < n ≤ 48                   | 40 < hash_index ≤ 48 |
```

Comment:
Jean changed this.... Not clear to me why :thinking:

Reply:
`PrivateCircuitPublicInputs` size increased from 41 to 45, so while hashing `PrivateCircuitPublicInputs`, we need 45 generators. We only allowed upto 44 items to be hashed before this, hence bumped it up to 48.